### ISSUE TITLE
egui_glow - no text being rendered

### ISSUE THREAD

**Issue by KnockerPulsar** [emilk/egui#1723]
**Describe the bug**
The UI body seems to render fine. No text shows up whatsoever.
![image](https://user-images.githubusercontent.com/12754772/172680083-5da0532e-633c-40bd-aed5-ae961550c193.png)

Here's the code it's supposed to be rendering:
```rust
  let needs_repaint = egui_glow.run(window.window(), |egui_ctx| {
                        egui::SidePanel::left("my_side_panel").show(egui_ctx, |ui| {
                            ui.label("Hello World!");
                        });
                    });
```
Another screenshot with code ripped straight from the example:
![image](https://user-images.githubusercontent.com/12754772/172682285-6b47511a-8dca-4aee-bbfb-aa9f962b0a45.png)

```rust
let needs_repaint = egui_glow.run(window.window(), |egui_ctx| {
                        egui::SidePanel::left("my_side_panel").show(egui_ctx, |ui| {
                            ui.heading("Hello World!");
                            if ui.button("Quit").clicked() {
                                quit = true;
                            }
                            ui.color_edit_button_rgb(&mut clear_color);
                        });
                    });
```

**To Reproduce**
1. Follow the initialization an main loop of [egui_glow's example](https://github.com/emilk/egui/tree/master/egui_glow). You might encounter a compilation error due to the example creating a `Arc::new(gl)` instead of `Rc::new(gl)`. Switch it out and you should be good to go.
2. Modify your `Cargo.toml` to be like this:
```rust
[package]
name = "egui_glow_test"
version = "0.1.0"
edition = "2021"


[dependencies]
image = "^0.24.2"
glutin = "*"

egui = { version = "0.18.0", default-features = false, features = [
  "bytemuck",
] }

egui_glow = {version = "0.18.0", features = [ "winit" ]}
bytemuck = "1.7"
glow = "0.11"
memoffset = "0.6"
tracing = "0.1"
```
Copied from the example's toml with a few modifications.
3. Try building and running the code. You should get an output window similar to the one above.

**Expected behavior**
Text similar to the examples should be drawn.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04

**Additional context**
A couple of other people reported this issue: 
- #1719 
- #327

**Comment by t18b219k** [comment]
it seems default font disabled in toml file.

**Comment by KnockerPulsar** [comment]
> it seems default font disabled in toml file.  

Ok, so I tried running the example from the repo's root by calling `cargo run --example pure_glow features="winit"` and it did run properly. I'm not sure however how to enable/disable the default font in my own project, any ideas?

**Comment by KnockerPulsar** [comment]
> enable egui default feature or add "default-font" to features .

So I've been trying to get the example to compile standalone since I forgot how I got egui_glow to compile in my project. For some reason, even with `cargo run --features="winit"`, `EguiGlow` isn't exposed to `egui_glow`. 
![image](https://user-images.githubusercontent.com/12754772/172849266-44dc7061-cdde-4354-adb3-9009a69d06d0.png)

To test this out, I created a new cargo binary project, copied the `pure_glow.rs` example to the main and copied over and modified the toml file. 

`main.rs` has a couple of extra lines at the top to declare used crates
```rust
extern crate glutin;
extern crate egui_glow;
```
The toml file is a bit more different, I've been playing around with it trying to get `EguiGlow` to be exposed. 
```toml
[package]
name = "egui_glow_test"
version = "0.18.1"
edition = "2021"
rust-version = "1.60"

[features]
default = ["winit"]

# for the winit integration:
# enable cut/copy/paste to os clipboard.
# if disabled a clipboard will be simulated so you can still copy/paste within the egui app.
clipboard = ["egui-winit?/clipboard"]

# for the winit integration:
# enable opening links in a browser when an egui hyperlink is clicked.
links = ["egui-winit?/links"]

# experimental support for a screen reader.
screen_reader = ["egui-winit?/screen_reader"]

# enable winit integration.
winit = ["egui-winit"]


[dependencies]
egui = { version = "0.18.0", default-features = true }

glow = "0.11"
memoffset = "0.6"
tracing = "0.1"

glutin = "0.28.0" # examples/pure_glow
egui_glow = "*"

egui-winit = { version = "0.18.0", optional = true, default-features = true }
```

I've even went into `egui_glow/lib.rs` and tried commenting the 2 lines that are supposed to enable/disable winit based on platform. I know this isn't the proper way to do it but I just wanted to get something to compile.
![image](https://user-images.githubusercontent.com/12754772/172849721-47a516ed-577d-4031-b68f-600d2dd1d226.png)

Any clues on what I might be missing? Sorry if I'm missing something obvious, cargo is quite new to me.

**Comment by KnockerPulsar** [comment]
So, I managed to get a standalone project to build. The issue related to `EguiGlow` was because I didn't turn on `winit` specifically for the `egui_glow` crate. This was done by adding a feature specifier as follows:
```
egui_glow = { version = "0.18.0", git = "https://github.com/emilk/egui/", features = ["winit"]}
```
You might notice the extra git specifier too, I just did that to match what the example was doing. I think using the crate from crates.io might be more stable due to fewer changes occurring there vs on this repo.

Here's currently the project's structure:
![image](https://user-images.githubusercontent.com/12754772/172962819-77f09b37-74a8-41e5-bf71-8ff732cfbb5b.png)

`main.rs` is the same as the `pure_glow` example.
The Cargo.toml file is also the same save for the addition of the egui_glow dependency mentioned at the start. I also made `egui` use this repo too. 
For anyone wondering why the example doesn't do this and the standalone has to, this is (at least I think so) because the example is part of the egui_glow crate, while the standalone project uses the crate so it has to specify it as a dependency explicitly.
